pkg/config/config.go

```diff
@@ -87,6 +87,34 @@ func newConfig(buildConfig []byte, rewriteDockerRepositoryName bool) (*Config, e
 		}
 	}
 
+	nexusAccess := NexusAccess{}
```

Comment:
Bør denne defaulte til gammel nexus?

Reply:
Nei, jeg tenker på NexusAccess som et statisk objekt.  Det gjenspeiler secreten som er lest opp, og så lenge vi bruker Nexus 2, så har jeg ikke lyst til å tukle med det. Architect bør skrives om igjen når Nexus 2 og aksess uten autorisasjon er ute av bildet.

Reply:
Men da må NexusUrl ut av typen. Nå sendes det to forskjellige URL'er til NexusDownloader.

Reply:
Nei, om det kommer en URL i NexusAccess, så settes mavenRepo slik at de er like. Vi må ta en diskusjon på dette med spesifikasjon av Nexus URL etterhvert.  Hardkodingen i Architect er etter min mening ikke pen.

Reply:
Har skrevet om kode slik at default Nexus URL nå settes på NexusAccess.  Fikset,